pyscript.core/tests/integration/test_script_type.py

```diff
@@ -81,12 +81,12 @@ def test_script_type_py_src_attribute(self):
         )
         assert self.console.log.lines[-1] == "hello from foo"
 
-    @pytest.mark.skip("FIXME: test failure is unrelated")
+    @pytest.mark.skip("FIXME: wait_for_pyscript is broken")
```

Comment:
I assume `wait_for_pyscript` is broken for the same execution-order reasons? Or is it more generally broken? Again, really just curious for later.

Reply:
> I assume wait_for_pyscript is broken for the same execution-order reasons? Or is it more generally broken? 

yes, it's the same problem.
The problem is that the current version of `wait_for_pyscript()` waits for `PyScript Ready` which is printed *before* the tags are executed, not after.

Basically right now all the tests are at risk of being flaky, but most of them are just fast enough that they work in practice.